include/main.h

```diff
@@ -1 +1,12 @@
+#ifndef __MAIN__H__
+#define __MAIN__H__
 
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <gsl/gsl_math.h>
+#include <gsl/gsl_eigen.h>
+
+#define SENSOR_COUNT 3
+
+#endif
```

Comment:
Point 13 of coding conventions suggests to have header guard like the exaple below:

#endif // __ MY_HEADER__